apps/api/src/hyp3_api/api-spec/openapi-spec.yml.j2

```diff
@@ -106,6 +117,11 @@ paths:
 components:
   schemas:
 
+    # TODO specify more information?
```

Comment:
I can take this TODO this afternoon

Reply:
:thinking: Hrmm, OpenAPI isn't good at describing schemas where we're using job types / parameter values as dictionary keys, e.g.:

```
foo:
  cost: 1.0
bar:
  cost: 1.0
another:
  cost_parameter: hello
  costs:
    a: 1.0
    b: 2.0
```

That makes lookups in the implementation easy, but makes describing what to expect via OpenAPI hard. If we use lists instead of dictionaries it makes describing the schema simple, but makes lookups at runtime more cumbersome:

```
- job_type: foo
  cost: 1.0
- job_type: bar
  cost: 1.0
- job_type: another
  cost_parameter: hello
  costs:
    - parameter_value: a
      cost: 1.0
    - parameter_value: b
      cost: 2.0
 ```

Punting this one and leaving it as-is for the moment.

Reply:
For posterity, we realized we could do this with Jinja, but not worth the effort.